### DISCUSSION TITLE
Plane traces

### DISCUSSION THREAD

**Discussion by ArianeNeves** [mtex-toolbox/mtex#1511] [Ask Anything]
Hi!
I'd like to plot the plane traces of {110} plane. Based on the Mtex forum, I write the code bellow:

% Planes
hh = Miller({1,1,0},{1,0,1},{0,1,1},{-1,1,0},{-1,0,1},{0,-1,1},ebsd.CS);
n110 = grain_subset.meanOrientation*hh;
n110.antipodal=1;

figure(1)
ipfKey =ipfTSLKey(ebsd)
plot(ebsd,ipfKey.orientation2color(ebsd.orientations))
hold on
plot(grains.boundary,'lineWidth',0.5,'lineColor','black')
for i = 1:length(n110)
hold on
quiver(grain_subset,cross(n110(:,i),zvector))
end

However, I noted differences plane traces between the MTEX and OIM. 

![Largura do bloco](https://user-images.githubusercontent.com/81368454/181765274-89568554-c7d7-4cc3-92e5-b746d931a44f.jpg)
OIM

![Largura do 2](https://user-images.githubusercontent.com/81368454/181765465-b9fcb56f-0ee6-4cbc-97fe-8710c7c38e79.jpg)
MTEX

Is there any error?

Thanks.

**Comment by kilir** [comment]
Hi,
maybe you did not load your data with the correct setting corresponding to the setup your data was acquired/exported on OIM?
Cheers,
Rüdiger

**Comment by ArianeNeves** [comment]
Thanks, Rüdiger!!!
